asab/config.py

```diff
@@ -48,6 +48,10 @@ class ConfigParser(configparser.ConfigParser):
 			"expiration": 60,
 		},
 
+		"asab:doc": {
+			"default_route_tag": "general"
```

Comment:
i'd choose `module_name` as a default. and possibly remove the `general` option completely, now that there are the better new options you implemented.